region/maridia/inner-green/East Sand Hall.json

```diff
@@ -507,6 +507,30 @@
         "A little higher run speed can also work but may require releasing jump near the peak to avoid getting caught on the overhang."
       ]
     },
+    {
+      "link": [2, 1],
+      "name": "East Sand Hall Cross-Room Blue Spring Ball Bounce (Right to Left)",
+      "notable": true,
+      "entranceCondition": {
+        "comeInWithBlueSpringBallBounce": {
+          "adjacentMinTiles": 18,
+          "adjacentBlueSpeedTiles": 15,
```

Comment:
At the lower speeds, I sometimes get stuck in the middle or don't make it out of the sand at the end.

Reply:
Yeah I think we wmay ant a way to express the minimum speed requirement directly, since the tiles requirements don't really make it clear.